auth/services/irma/validator.go

```diff
@@ -83,9 +84,51 @@ type VPProof struct {
 	ProofValue string `json:"proofValue"`
 }
 
+type irmaVPVerificationResult struct {
+	validity            contract.State
+	vpType              contract.VPType
+	disclosedAttributes map[string]string
+	contractAttributes  map[string]string
+}
+
+func (I irmaVPVerificationResult) Validity() contract.State {
+	return I.validity
+}
+
+func (I irmaVPVerificationResult) VPType() contract.VPType {
+	return I.vpType
+}
+
+func (I irmaVPVerificationResult) DisclosedAttribute(key string) string {
```

Comment:
same for this (check demos).

Also: this is not according to RFC002:

> pbdf.gemeente.personalData.initials
pbdf.gemeente.personalData.familyname
pbdf.gemeente.personalData.dateofbirth
pbdf.gemeente.personalData.digidlevel
pbdf.sidn-pbdf.email.email

https://nuts-foundation.gitbook.io/drafts/rfc/rfc002-authentication-token#7-1-irma

Reply:
I'll make an issue for that, since it's separate from this PR

Reply:
https://github.com/nuts-foundation/nuts-node/issues/508